### PR TITLE
Add equals methods to food items and stations

### PR DESCRIPTION
**Pull request by ilikesymmetry**
This will fix the merge_events method so we stop getting duplicate menus at Mattins (for example)

### CODE REVIEW

File: src/data.py [hunk 1]
```diff
@@ -296,8 +296,9 @@ def merge_hours(eateries):
       if len(operating_hour.events) <= 1:  # ignore hours that don't have multiple events
         continue
       base_event = operating_hour.events[0]
-      for event in operating_hour.events[:]:  # iterate over copy of list so we can safely remove
-        if event.start_time == base_event.end_time and not event.menu:
+      for event in operating_hour.events[1:]:  # iterate over copy of list so we can safely remove
+        if event.start_time == base_event.end_time and \
```
Comment (young-k) [inline src/data.py:300]: prefer:

```python
if (event.start_time == base_event.end_time and
    event.menu and
    not event.menu[0].equals(base_event.menu[0])):
  # code here
```

**Review by young-k — APPROVED** [review]
LGTM

**Review by manishsaha — APPROVED** [review]
Looks good to me!

**Review by meganle — APPROVED** [review]
lgtm!

**Review by orkosinha — APPROVED** [review]
lgtm!

**Pull request merged**